plugins/builds/index.js

```diff
@@ -873,16 +1020,44 @@ const buildsPlugin = {
                         return createInternalBuild(internalBuildConfig);
                     }
 
-                    if (!['CREATED', null, undefined].includes(existNextBuild.status)) {
+                    if (!['CREATED', null, undefined].includes(existNextBuild.status) && !current.stage) {
```

Comment:
As suggested here
https://github.com/screwdriver-cd/screwdriver/pull/2870/files/9f679937feaa4e278d80b3b1ef245984c9b03993#r1448111687, we need to revert this change.

```suggestion
                    if (!['CREATED', null, undefined].includes(existNextBuild.status)) {
```